sdk/log/logger_test.go

```diff
@@ -18,6 +19,14 @@ import (
 )
 
 func TestLoggerEmit(t *testing.T) {
+	nowSwap := now
+	t.Cleanup(func() {
+		now = nowSwap
+	})
+	now = func() time.Time {
+		return time.Date(2010, time.January, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
This should be declared as a variable in the parent function scope so it can be used here and below in the check.